source/documentation/index.md

```diff
@@ -110,36 +110,99 @@ See some examples below.
+<table>
+<tr>
+    <th>Title</th>
+    <th>Description</th>
+</tr>
+<tr>
+    <td><a href="https://github.com/hmrc/transits-movements-trader-at-departure/blob/master/conf/xsd/CC009A.xsd">Cancellation decision message CC009A (IE009)</a></td>
+    <td>Message type received by the trader from the office at departure in reference to a cancellation request (IE014)</td>
+</tr>
+<tr>
+    <td><a href="https://github.com/hmrc/transits-movements-trader-at-departure/blob/master/conf/xsd/CC016A.xsd">Declaration rejection message CC016A (IE016)</a></td>
+    <td>Message type received from the office at departure to the trader at departure rejecting a Declaration data message (IE015)</td>
+</tr>
+<tr>
+    <td><a href="https://github.com/hmrc/transits-movements-trader-at-departure/blob/master/conf/xsd/CC028A.xsd">MRN allocated message CC028A (IE028)</a></td>
+    <td>Message type received from the office at departure to the trader at departure allocating a Movement Reference Number (MRN)</td>
+</tr>
+<tr>
+    <td><a href="https://github.com/hmrc/transits-movements-trader-at-departure/blob/master/conf/xsd/CC029B.xsd">Goods released for transit CC029B (IE029)</a></td>
+    <td>Message type received from the office at departure to the trader at departure to say the goods are released for transit</td>
+</tr>
+<tr>
+    <td><a href="https://github.com/hmrc/transits-movements-trader-at-departure/blob/master/conf/xsd/CC045A.xsd">Write-off notification CC045A (IE045)</a></td>
+    <td>Message type received from the office at departure to the trader at departure to say that the transit movement is finished</td>
+</tr>
+<tr>
+    <td><a href="https://github.com/hmrc/transits-movements-trader-at-departure/blob/master/conf/xsd/CC051B.xsd">No release for transit CC051B (IE051)</a></td>
+    <td>Message type received from the office at departure to the trader at departure to say that the movement cannot be released for transit</td>
+</tr>
+<tr>
+    <td><a href="https://github.com/hmrc/transits-movements-trader-at-departure/blob/master/conf/xsd/CC055A.xsd">Guarantee not valid CC055A (IE055)</a></td>
+    <td>Message type received from the office at departure to the trader at departure to say that their guarantee is not valid</td>
+</tr>
+<tr>
+    <td><a href="https://github.com/hmrc/transits-movements-trader-at-departure/blob/master/conf/xsd/CC060A.xsd">Control decision notification CC060A (IE060)</a></td>
+    <td>Message type received from the office at departure to tell the trader at departure they wish to carry out a control of the goods</td>
+</tr>
+<tr>
+    <td><a href="https://github.com/hmrc/transits-movements-trader-at-departure/blob/master/conf/xsd/CC928A.xsd">Declaration received CC928A (IE928)</a></td>
+    <td>Message type received from NCTS of a positive acknowledgement of a departure declaration message (IE015)</td>
+</tr>
+</table>
+
 
 ###Clarifications and omissions
```

Comment:
H2

Reply:
New H2 title for Postman and cURL snippets - "Example XML Requests"

Reply:
API Features - make H2

Reply:
Replace Default Guarantee Instruction, Rate Limits and Data Cap with Bold Text, with sub-bullets becoming paragraphs